app/internal/page/login.go

```diff
@@ -0,0 +1,18 @@
+package page
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/signin"
+)
+
+func Login(c signin.Client, appPublicURL, clientID, signInBaseURL string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		redirectURL := fmt.Sprintf("%s%s", appPublicURL, c.AuthCallbackPath)
```

Comment:
might be nicer to construct the `redirectURL` in `main.go` then pass it down to the places needed, would also allow you to remove the `AuthCallbackPath` from `Client`